source/hardware/controllers/denon_mc7000.rst

```diff
@@ -316,6 +329,9 @@ Use :hwlabel:`SHIFT` + pad button to stop a sampler while playing or eject a sam
    :alt: Denon MC7000 SAMPLER Mode
    :figclass: pretty-figures
 
+.. hint::
+   To multi play triggered samplers at the same time you can set the user variable ``MC7000.prevSamplerStop`` inside the :ref:`JavaScript file<denon_mc7000_uservariables>` to ``false``.
```

Comment:
```suggestion
   To use default Mixxx behaviour and allow playing multiple samplers at the same time you can set the user variable ``MC7000.prevSamplerStop`` inside the :ref:`JavaScript file<denon_mc7000_uservariables>` to ``false``.
```

Reply:
That's it from me. LGTM when the line above is changed!
Someone who reviewed the actual mapping should take a look though.

Reply:
done